tests/serve/plugin.py

```diff
@@ -72,17 +66,29 @@ def writer_iterator():
 
             stub.Write(writer_iterator())
 
-            response = stub.GetTables(plugin_pb2.GetTables.Request(tables=["*"]))
+            response = stub.GetTables(plugin_pb2.GetTables.Request(tables=["*"],skip_tables=[]))
             schemas = arrow.new_schemas_from_bytes(response.tables)
-            assert len(schemas) == 4
+            assert len(schemas) == 3
 
-            response = stub.Sync(plugin_pb2.Sync.Request(tables=["*"]))
+            response = stub.Sync(plugin_pb2.Sync.Request(tables=["*"],skip_tables=[]))
+            total_migrate_tables = 0
             total_records = 0
+            total_errors = 0
             for msg in response:
-                if msg.insert is not None:
+                message_type = msg.WhichOneof("message")
```

Comment:
`msg.WhichOneof("message")` is the correct way to know the message type.
On other non insert message the condition `msg.insert is not None` is true as well